package.json

```diff
@@ -176,6 +179,7 @@
         "webpack-node-externals": "^3.0.0"
     },
     "dependencies": {
+        "@esbuild-plugins/node-resolve": "^0.1.4",
```

Comment:
All of the new deps are devDependencies, not prod dependencies, no?

Reply:
Good catch! I must have missed the appropriate flag when adding all my changes to this branch.